src/common.rs

```diff
@@ -10,7 +10,7 @@ pub trait AsU32 {
 
 cfg_if::cfg_if! {
     if #[cfg(any(windows, target_os = "unknown", target_arch = "wasm32"))] {
-        /// Process id.
+        #[doc = include_str!("../../md_doc/pid.md")]
```

Comment:
```suggestion
        #[doc = include_str!("../md_doc/pid.md")]
```

You can test it by running `cargo doc`.